rundown/resources/sportsbook.py

```diff
@@ -0,0 +1,7 @@
+from pydantic import BaseModel
+
+
+class Sportsbook(BaseModel):
+    affiliate_name: str
```

Comment:
it would be great to provide a way to extract specific sportsbooks by name instead of just the `affiliate_id`. For example, `sportsbooks.pinnacle` or similar.

Reply:
> What would be the use case of doing this?

It's to remember the names of the sportsbooks rather than their identifier